src/ngx_http_lua_ssl_certby.c

```diff
@@ -1319,6 +1319,21 @@ ngx_http_lua_ffi_set_priv_key(ngx_http_request_t *r,
 }
 
 
+int
+ngx_http_lua_ffi_get_ssl_pointer(ngx_http_request_t *r,
```

Comment:
We are moving to the meta-lua-nginx-module:

https://github.com/openresty/meta-lua-nginx-module

So this change deserves supporting other subsystems as well (like the `stream` subsystem) if we indeed want to have this.

Reply:
Won't this get added when the stream subsystem gains `ssl_certificate_by_lua_block` support?